websites/V/VRV/dist/metadata.json

```diff
@@ -13,11 +13,12 @@
 	"service": "VRV",
 	"description": {
 		"en": "VRV is the home of your favorite channels, events, and communities celebrating anime, animation, video games, comics, science fiction, fantasy, and tech.",
+		"fr": "VRV est l'endroit pour vos chaînes, événements and communautés favoris autour des animes, de l'animation, des jeux videos, des comics, de la science fiction, de la fantasie et de la technologie.",
 		"nl": "VRV is de thuisbasis van je favoriete kanalen, evenementen en communities die anime, animatie, videogames, strips, science fiction, fantasy en tech vieren.",
 		"ga_IE": "Is é VRV baile na gcainéal, na n-imeachtaí agus na bpobal is fearr leat ag ceiliúradh anime, beochana, cluichí físeáin, greannáin, ficsean eolaíochta, fantaisíochta agus ardteicneolaíochta."
 	},
 	"url": "vrv.co",
-	"version": "2.0.0",
+	"version": "2.1.0",
```

Comment:
```suggestion
	"version": "2.0.1",
```

Reply:
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8